lib/writer.js

```diff
@@ -6,6 +6,7 @@ var DEFAULT_TYPE = 'other';
 var PR_REGEX = new RegExp(/#[1-9][\d]*/g);
 var TYPES = {
   build: 'Build System / Dependencies',
+  breaking: 'Breaking Changes',
```

Comment:
It goes the alphabetical sorting of the others, but I think that breaking changes should probably be at the top of the changelog so that they are more highlighted.

Reply:
👍